raiden/exceptions.py

```diff
@@ -255,13 +280,20 @@ class InsufficientGasReserve(RaidenError):
 
 
 class BrokenPreconditionError(RaidenError):
-    """ Raised while checking transaction preconditions
-    which should be satisfied before sending the transaction.
-    This exception when:
-    1. An assert or a revert in the smart contract would be hit for
-    triggering block.
+    """ Raised when the chain doesn't satisfy transaction preconditions
+    that proxies check at the specified block.
+
+    This exception should be used, when the proxiy already sees
```

Comment:
poxiy -> proxy 

already sees -> already sees that  (?)

Reply:
Fixed both.